src/app.js

```diff
@@ -20,6 +20,7 @@ import { openapi } from 'utils/load-openapi';
 import { validateDataSource } from 'utils/validate-data-source';
 
 const serverConfig = config.get('server');
+const { version } = openapi.info;
```

Comment:
Can you also destructure `title`? https://github.com/osu-mist/express-api-skeleton/blob/2ca2ecf649546597cbe856bc39930a4d19ae1903/src/app.js#L87